examples/cloudngfw_isolated_design/example.tfvars

```diff
@@ -12,7 +12,7 @@ global_tags = {
   Owner       = "PS Team"
 }
 
-ssh_key_name = "ssh_key" # TODO: update here
+ssh_key_name = "example-ssh-key" # TODO: update here

```

Comment:
On account used by GitHub workflows I don't see key pair `example-ssh-key` in region `us-east-1` - we need to use existing key pair name or add Terraform code to create it programatically.